use-cases/using-Intel-QAT-and-kata.md

```diff
@@ -0,0 +1,484 @@
+
+[Intel QuickAssist Crypto Poll Mode Driver](https://dpdk-docs.readthedocs.io/en/latest/cryptodevs/qat.html)
+
+## Steps to enable Intel QAT in Kata Containers
+
+Some of the steps only have to be done once. Some have to be done every 
+reboot. Other steps are required only when the host kernel changes.
+
+## Script variables 
+
+The variables listed below must be set before running through the scripts. 
+These variables refer to locations to store modules and configuration files on
+ the host and as well as links to the drivers to use. Modify these as needed 
+ to point to updated drivers or different install locations.
+
+### Set environment variables (Every Reboot)
+
+Make sure to check [01.org](https://01.org/intel-quickassist-technology) for 
+the latest driver.
+
+```sh
+$ export QAT_DRIVER_VER=qat1.7.l.4.5.0-00034.tar.gz
+$ export QAT_DRIVER_URL=https://01.org/sites/default/files/downloads/${QAT_DRIVER_VER}
+$ export QAT_CONF_LOCATION=~/QAT_conf
+$ export QAT_DOCKERFILE=https://raw.githubusercontent.com/intel/intel-device-plugins-for-kubernetes/master/demo/openssl-qat-engine/Dockerfile
+$ export QAT_SRC=~/src/QAT
+$ export GOPATH=~/src/go
+$ export OSBUILDER=~/src/osbuilder
+$ export KATA_KERNEL_LOCATION=~/kata
+$ export KATA_ROOTFS_LOCATION=~/kata
+```
+
+## Prepare the Clear Linux Host
+
+The host could be a bare metal instance or a virtual machine. If using a 
+virtual machine, make sure that KVM nesting is enabled.  All of the 
+instructions below reference an Intel QAT.  Some of the instructions must be 
+modified if using a different Intel QAT device. You can identify the Intel QAT
+chipset by executing the following.
+
+### Identify which PCI Bus the Intel QAT card is on
+
+```sh
+$ for i in 0434 0435 37c8 1f18 1f19; do lspci -d 8086:$i; done
+```
+
+### Install necessary bundles for Clear Linux and disable kernel updates
+
+The Clear Linux installer generally installs the `os-core`, `os-core-update`, 
+and `openssh-server` bundle. Add the following minimum bundles including the 
+Linux kernel headers to complete the rest of this guide. The Linux kernel 
+headers are needed to compile the out of tree kernel modules to support Intel 
+QAT on the host. The `os-clr-on-clr` is a large development bundle that can be 
+installed instead to include everything below with additional items. This 
+guide installs the Linux LTS kernel version `4.19` from 2018 because the 
+crypto modules are set to `m` and not `y` which allows them to be replaced 
+with the out of tree driver. Otherwise you will need to build your own Clear 
+Linux kernel with the QAT drivers compiled in.
+
+```sh
+$ sudo swupd autoupdate --disable
+$ sudo swupd bundle-add network-basic kernel-lts2018 linux-lts-dev linux-dev make c-basic go-basic containers-virt dev-utils devpkg-elfutils devpkg-systemd
+$ sudo systemctl start docker
+$ sudo systemctl enable docker
+$ sudo -E gpasswd -a $USER docker
```

Comment:
I would avoid adding this as it is not a good practice from security point of view.

Reply:
Do you think I should then change all my docker commands to have sudo?  That is why I did this.

Reply:
Never mind. I see in your comments further down to do just this. I'll update.